.github/renovate.json

```diff
@@ -26,6 +26,7 @@
         "^envalid$",
         "^execa$",
         "^lit$",
+        "^typescript$",
```

Comment:
Allows updates of TypeScript in `block-scripts` too (it’s listed in `dependencies` rather than `devDependencies` there)